ExploreASL_Master.m

```diff
@@ -71,23 +71,33 @@
     
     % -----------------------------------------------------------------------------
     % Import Master
-    x = ExploreASL_ImportMaster(x);
+    if x.opts.bImportData
+        x = ExploreASL_ImportMaster(x);
+    end
+    % Store logging information about errors/warnings in backup variable
+    if isfield(x,'logging')
+    	loggingBackUp = x.logging;
+    end
     
     % -----------------------------------------------------------------------------
     % Re-Initialize for potential data loading/processing
-    if x.bReinitialize > 0
-        x = ExploreASL_Initialize(x.DataParPath, x.ImportModules, x.ProcessModules, x.bPause, x.iWorker, x.nWorkers);
+    if x.opts.bReinitialize > 0
+        x = ExploreASL_Initialize(x.opts.StudyRoot, x.opts.ImportModules, x.opts.ProcessModules, x.opts.bPause, x.opts.iWorker, x.opts.nWorkers);

```

Comment:
These parameters have the same name in `ExploreASL_Initialize right`?

Reply:
They should...